fluster/main.py

```diff
@@ -330,6 +330,13 @@ def _add_download_cmd(self, subparsers: Any) -> None:
             "files such as .zip, .tar.gz, etc",
             action="store_true",
         )
+        subparser.add_argument(
+            "-r",
+            "--retries",
+            help="number of retries, before failing",
```

Comment:
I think the name `retries` is too generic and can lead to confusion about what is being retried:  it could be failing tests for example.
I think `download-retries` would be a better name.